<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVolunteerName</class>
 <widget class="QMainWindow" name="AddVolunteerName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volunteer Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="eventComboBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>574</width>
         <height>432</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="volunteerLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveVolunteers">
      <property name="text">
       <string>Save Volunteers</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
